### DISCUSSION TITLE
Have any have native lib issue (Android application bundle)

### DISCUSSION THREAD

**Discussion by allenhsu-17live** [google/filament#4477] [Q&A]
Hi, I got some native crash issue from google play console, the detail like this
```
 #00  pc 00000000003f5af0  /data/app/com.machipopo.media17-cwa4a9Pk662mteWjEsRAnw==/split_config.arm64_v8a.apk!lib/arm64-v8a/libgltfio-jni.so (offset 0x196e000)
```
 
 I found some information relate this, 
 
 https://stackoverflow.com/questions/62807988/android-app-bundle-only-native-library-failed-to-load
 And I found out for libfilament will exclude JNI Lib, like this
```
// No need to package up the following shared libs, which arise as a side effect of our
    // externalNativeBuild dependencies. When clients pick and choose from project-level gradle
    // dependencies, these shared libs already get pulled in, so we need to avoid the error:
    // "More than one file was found with OS independent path ..."
    packagingOptions {
        exclude 'lib/*/libfilament-jni.so'
        exclude 'lib/*/libgltfio-jni.so'
    }
```    

filament to exclude jni.so ? Have anyone can help this question?
I guest the exclude can't fit Google AAB ? And how to fix it?

**Comment by romainguy** [comment]
The error shows you have the library. Could you please share the full logs?

**Reply by allenhsu-17live** [reply]
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 0, tid: 0 >>> com.machipopo.media17 <<<
```
backtrace:
  #00  pc 000000000041c0bc  /data/app/~~fM4zW4DMdtz8B5Fe9qZ07g==/com.machipopo.media17-lzk0VCjonM0sNykU5sI53w==/split_config.arm64_v8a.apk!lib/arm64-v8a/libgltfio-jni.so (offset 0x1942000)
```

That's all...
